kodilla-stream/src/main/java/com/kodilla/stream/StreamMain.java

```diff
@@ -12,7 +12,7 @@ public static void main(String[] args) {
         Date limitDate = new Date();
 
         Map<Integer, ForumUser> theResultMapOfForumusers = forum.getUserList().stream()
-                .filter(forumUser -> forumUser.getBd().getYear() < 2000)
+                .filter(forumUser -> forumUser.getBd().getYear() <= 2000)
```

Comment:
To nadal nie jest to :) Urodzony 11.11.2000 nie ma jeszcze 20 lat, a u ciebie przejdzie.